spec/values/calculation/acos.hrx

```diff
@@ -0,0 +1,185 @@
+<===> negative_one/output.css
+a {
+  b: 180deg;
+}
+
+<===>
+================================================================================
+<===> zero/input.scss
+a {b: acos(0)}
+
+<===> zero/output.css
+a {
+  b: 90deg;
+}
+
+<===>
+================================================================================
+<===> zero_fuzzy/input.scss
+a {b: acos(0.000000000001)}
+
+<===> zero_fuzzy/output.css
+a {
+  b: 89.9999999999deg;
+}
+
+<===>
+================================================================================
+<===> one/input.scss
+a {b: acos(1)}
+
+<===> one/output.css
+a {
+  b: 0deg;
+}
+
+<===>
+================================================================================
+<===> one_fuzzy/input.scss
+a {b: acos(1.000000000001)}
+
+<===> one_fuzzy/output.css
+a {
+  b: calc(NaN * 1deg);
+}
+
+<===>
+================================================================================
+<===> greater_than_one/input.scss
+a {b: acos(2)}
+
+<===> greater_than_one/output.css
+a {
+  b: calc(NaN * 1deg);
+}
+
+<===>
+================================================================================
+<===> operation/simplification/input.scss
+a {
+  b: acos(3px - 1px + var(--c));
+}
+
+<===> operation/simplification/output.css
+a {
+  b: acos(2px + var(--c));
+}
+
+<===>
+================================================================================
+<===> preserved/variable/input.scss
+a {
+  b: acos(var(--c));
+}
+
+<===> preserved/variable/output.css
+a {
+  b: acos(var(--c));
+}
+
+<===>
+================================================================================
+<===> preserved/operation/percentage/input.scss
+a {b: acos(17px + 5%)}
+
+<===> preserved/operation/percentage/output.css
+a {
+  b: acos(17px + 5%);
+}
+
+<===>
+================================================================================
+<===> error/units/input.scss
+a {b: acos(1px)}
+
+<===> error/units/error
+Error: Expected 1px to have no units.
+  ,
+1 | a {b: acos(1px)}
+  |       ^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/type/input.scss
+a {b: acos("0")}
+
+<===> error/type/error
+Error: Expected number, variable, function, or calculation.
+  ,
+1 | a {b: acos("0")}
+  |            ^
+  '
+  input.scss 1:12  root stylesheet
+
+<===>
+================================================================================
+<===> error/zero_args/input.scss
+a {b: acos()}
+
+<===> error/zero_args/error
+Error: Expected number, variable, function, or calculation.
+  ,
+1 | a {b: acos()}
+  |            ^
+  '
+  input.scss 1:12  root stylesheet
+
+<===>
+================================================================================
+<===> error/too_many_args/input.scss
+a {b: acos(0, 0)}
+
+<===> error/too_many_args/error
+Error: expected "+", "-", "*", "/", or ")".
+  ,
+1 | a {b: acos(0, 0)}
+  |             ^
+  '
+  input.scss 1:13  root stylesheet
```

Comment:
Also test too few arguments.

Reply:
Since it's a single argument function, isn't it enough with the `too_few_args` test that's [here](https://github.com/sass/sass-spec/blob/0e3fef7a107f236920e480e574c01880a9cf4bb7/spec/values/calculation/acos.hrx#L110)?